src/swarm-lang/Swarm/Language/Parser/Lex.hs

```diff
@@ -163,7 +163,7 @@ primitiveTypeNames = "Cmd" : baseTypeNames
 
 -- | List of keywords built into the language.
 keywords :: [Text]
-keywords = T.words "let in def tydef end true false forall require requirements"
+keywords = T.words "let in def tydef end true false forall require requirements rec"
```

Comment:
This should probably be followed up with updating the editor configurations (`swarm-mode.el`, `swarm.vim` etc)?

Reply:
Good catch.  I will update that shortly.